course/src/isometry.cc

```diff
@@ -0,0 +1,102 @@
+#include <cmath>
+#include <cstdint>
+#include "isometry.h"
+
+namespace ekumen {
+namespace math {
+
+Vector3::Vector3(const double& x, const double& y, const double& z) : x_(x), y_(y), z_(z) {}
+
+Vector3::~Vector3() {}
+
+// Class constants
+const Vector3 Vector3::kUnitX = Vector3(1., 0., 0.);
+const Vector3 Vector3::kUnitY = Vector3(0., 1., 0.);
+const Vector3 Vector3::kUnitZ = Vector3(0., 0., 1.);
+const Vector3 Vector3::kZero = Vector3(0., 0., 0.);
+
+Vector3 Vector3::operator+(const Vector3& vec) const {
+    return Vector3(this->x() + vec.x(), this->y() + vec.y(), this->z() + vec.z());
```

Comment:
Reimplement this function using `+=`.

Reply:
Same for subtraction and product.

Reply:
Just in case, I am using it in += operator instead to do that.